omero/developers/Modules/Search.rst

```diff
@@ -39,7 +39,16 @@ Queries
 Search queries are very similar to Google searches. When search terms
 are entered without a prefix ("name:"), then the default field will be
 used which combines all available fields. Otherwise, a prefix can be
-added to restrict the search.
+added to restrict the search. The search terms will be by default parsed as if there was "OR" operator
+between them (see example in the "Indexing" paragraph). 
+Further, tokenizing happens on all non-alphanumerical signs, such as underscore, 
+hyphen etc. Again, when the term is tokenized, 
+the default parsing of the tokens is with "OR" operator between the tokens. 
```

Comment:
Hmmm - I wonder if this paragraph needs to separate out the tokenizing from the querying...
The tokenizing is just all "non-alpha-numerical characters" (not signs).
The querying is about combing the tokens with "AND" and "OR", but I think it's confusing to mention "OR" in the same sentence as tokenising, since tokenising is independent of operators?